be/src/vec/functions/function_string.h

```diff
@@ -540,6 +542,11 @@ class FunctionStringConcatWs : public IFunction {
             } else {
                 null_list[i] = &const_null_map->get_data();
             }
+
+            if (is_array(remove_nullable(block.get_by_position(arguments[i]).type))) {
```

Comment:
also can use check_column<ColumnArray>, then we do not need to consider nullable again.

Reply:
done